snuba/admin/notifications/slack/utils.py

```diff
@@ -38,6 +44,18 @@ def build_runtime_config_text(data: Any, action: AuditLogAction) -> Optional[str
         return None
 
 
+def build_migration_run_text(
+    data: Any, user: str, action: AuditLogAction
+) -> Optional[str]:
+    if action == AuditLogAction.RAN_MIGRATION:
+        action_text = f":runner: ran migration {data['migration']}"
+    elif action == AuditLogAction.REVERSED_MIGRATION:
+        action_text = f":uno-reverse: reversed migration {data['migration']}"
+    else:
+        return None
+    return f":warning: *Migration:* user '{user}' {action_text}  , force={data['force']}, fake={data['fake']}"
```

Comment:
do you mind just adding like a couple new lines after the *Migration:* just so its a tad easier to read? 

This is what we working with: 
![Screen Shot 2022-11-23 at 10 29 41 AM](https://user-images.githubusercontent.com/15368179/203621977-d924bd9c-087b-4c68-8d27-8c258bc978f1.png)

otherwise I think it looks good!

Reply:
also actually we do already put the user at the footer if you wanted to leave that out in the text above

locally it shows `by <>` but it will be `by accounts.google.com:meredith@sentry.io`

Reply:
Made some changes. It should look better now.